specs/signature-envelope-jws.md

```diff
@@ -167,13 +171,13 @@ Since Notary v2 restricts one signature per signature envelope, the compliant si
 
 ### Supported `alg` header values
 
-Notary v2 implementation MUST enforce the following constraints on signature generation and verification:
+Notary Project specification implementation MUST enforce the following constraints on signature generation and verification:
```

Comment:
The Notary Project specification implementation MUST enforce ...

Reply:
Update it to `Implementation of the Notary Project specification`